tools/cldr-apps/js/src/views/MainHeader.vue

```diff
@@ -188,7 +188,7 @@ header {
   padding: 0.2em;
   background-color: white;
   background-image: linear-gradient(white, #e7f7ff);
-  z-index: 10000000000; /* prevent transparency when scrolling; also stay in front of "overlay"  */
+  z-index: 900; /* prevent transparency when scrolling; also stay in front of "overlay"  */
 }
```

Comment:
Do we have a list anywhere of the z-index values used by ST, either directly or through libraries? I think I copied the "10000000000" from whatever code predates this, and while it does seem extreme, how do we know 900 is enough to stay in front?

Reply:
Findable by searching the code. 

This will improve as bootstrap is gone and all of our views are through vue.